data/vrishti29.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Vrishti Garg",
+  "bio": "I am a designer"
```

Please add the missing comma here.

```suggestion
  "bio": "I am a designer",
```